### PR TITLE
@dzucconi -> Artist changes

### PR DESCRIPTION
**Pull request by 1aurabrown**
Added cover image to sale, extracted and updated sale sort.

Also wasn't sure how to resolve a sale's image, since it wasn't contained in it own nested field, but rather all the image fields are flattened into the sale json. Supplying just the necessary fields seems to work, and then I wondered why we don't do the same for partner shows, so I made that change too.

### CODE REVIEW

File: schema/sales.js [hunk 1]
```diff
@@ -1,44 +1,12 @@
 import gravity from '../lib/loaders/gravity';
 import Sale from './sale';
+import SaleSorts from './sorts/sale_sorts';
```
Comment (dzucconi) [inline schema/sales.js]: I think it'd make more sense, organizationally if this was in /sale/sorts.js and sale.js => sale/index.js

File: schema/partner_show.js [hunk 2]
```diff
@@ -100,12 +100,7 @@ const PartnerShowType = new GraphQLObjectType({
     },
     cover_image: {
       type: Image.type,
-      resolve: ({ id }, options) => {
-        return gravity(`partner_show/${id}/default_image`, options)
-          .then(Image.resolve)
-          // Shows without a default_image reject with a 404
-          .catch(() => null);
-      },
+      resolve: ({ image_versions, image_url }) => Image.resolve({ image_versions, image_url }),
```
Comment (dzucconi) [inline schema/partner_show.js]: TIL we had an image already

File: schema/sales.js (hunk 1 above)
Comment (1aurabrown) [inline schema/sales.js]: sure, whatever, just saw that there was a sorts directory

File: schema/partner_show.js (hunk 2 above)
Comment (1aurabrown) [inline schema/partner_show.js]: ¯\_(ツ)_/¯

**Comment by 1aurabrown** [comment]
merging  w damon's ok

**Pull request merged**